src/main/java/in/projecteka/consentmanager/link/hip_link/model/request/Requester.java

```diff
@@ -0,0 +1,9 @@
+package in.projecteka.consentmanager.link.hip_link.model.request;
+
+import lombok.Value;
+
+@Value
+public class Requester {
+    Integer id;
+    String type;
```

Comment:
You can make this ENUM - there are only 2 types of requesters. You can just start with HIP (as the use case is only for Link right now)

Reply:
done